lib/bitstyles_phoenix/components/use_svg.ex

```diff
@@ -0,0 +1,47 @@
+          </g>
+        </svg>
+      </symbol>
+    </svg>
+    """
+  )
+
+  story(
+    "A referenced SVG (external)",
+    """
+        iex> safe_to_string ui_svg("logo", external: "assets/icons.svg")
```

Comment:
locally this does not work in the docs, because it accesses `file://`, but maybe it works in the hosted version 🤷